packages/site/src/data/vue/courses.ts

```diff
@@ -13,20 +13,21 @@ export const courseTags = [
 	"SCSS",
```

Comment:
Are we replacing "SCSS" with "sass"? If so, this one should be removed (since sass is added at the end)